### ISSUE TITLE
Rule proposal - Directive values should always be passed using brackets

### ISSUE THREAD

**Issue by luiscla27** [angular-eslint/angular-eslint#858]
**This rule proposal is related to an issue**, the new rule would be read like this:
_Every directive declared in the current NgModule, should always be used using brackets_

**Proposal**
Currently, you can do this:
```html
<button SomeCustomDirective="5" />
```
The expected would be this:
```html
<button [SomeCustomDirective]="5" />
```

**Description of the issue**

Without the brackets, the IDE doesn't have a way of knowing which variable type is expecting, so it's easy to sometimes pass a wrong value... especially when the directive is refactored (let's say the expected value is `Array<number>` due to refactorization changes, but before it was `number`) 

In my case, our team separated a bunch of angular components, each one to a single own module so we were able to call them lazy. And we didn't realize that there it was a lot of missing module imports from NgModule's until we ran the app. I think our case counts as refactorization.

**Feedback**
Please guide me if this is not the right place to ask for this new rule request. I posted in this repo because someone in the `eslint`  project advised me to report this feature request in the **Angular-specific ESLint plugin** ([see this issue](https://github.com/eslint/eslint/issues/14326))

**Versions**

| package                                   | version |
| ----------------------------------        | ------- |
| `@angular-eslint/eslint-plugin-template`  | `13.0.1` |
| `@angular-eslint/template-parser`         | `13.0.1` |
| `@typescript-eslint/parser`               | `5.6.0` |
| `ESLint`                                  | `8.4.1` |
| `node`                                    | `16.13.1.0` |

```sh
# Please run `npx ng version` in your project and paste the full output here:
Angular CLI: 13.1.0
Node: 16.13.1
Package Manager: npm 8.1.2
OS: win32 x64

Angular: 13.1.0
... animations, cli, common, compiler, compiler-cli, core
... elements, forms, language-service, platform-browser  
... platform-browser-dynamic, router, service-worker     

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1301.0
@angular-devkit/build-angular   13.1.0
@angular-devkit/core            13.1.0
@angular-devkit/schematics      13.1.0
@schematics/angular             13.1.0
rxjs                            7.4.0
typescript                      4.4.4

```
(this whole rule request applies also for for `@Input` values)

**Comment by luiscla27** [comment]
Closed in favor of the following discussion:
https://github.com/angular-eslint/angular-eslint/discussions/860
